sdk/storage/test-resources-post.ps1

```diff
@@ -172,3 +172,6 @@ $content | Set-Content $TestConfigurationPath
 Write-Verbose "Setting AZ_STORAGE_CONFIG_PATH environment variable used by Storage Tests"
 # https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/authoring/commands.md#logging-commands
 Write-Host "##vso[task.setvariable variable=AZ_STORAGE_CONFIG_PATH]$TestConfigurationPath"
+
+# Wait until RBAC replicates. It has 5min SLA. https://github.com/Azure/azure-sdk-for-net/issues/17384 to find better solution.
+Start-Sleep -s 90

```

Comment:
It would be helpful to log that we're sleeping for 90 seconds too, that way someone reading the log output trying to figure out what's taking so long knows what's going on (especially if we bump the timeout).